packages/client/src/client.ts

```diff
@@ -80,6 +80,6 @@ export class Client extends gen.Client implements types.IClient {
       throw new errors.UploadFileError(`Failed to upload file: ${err.message}`, <AxiosError>err, file)
     }
 
-    return await this.getFile({ id: file.id })
```

Comment:
note: Calling getFile as part of uploadFile isn't needed anymore because upsertFile now includes the URL of the file in its response.